test/hotspot/jtreg/vmTestbase/nsk/jvmti/SetJNIFunctionTable/setjniftab001/setjniftab001.cpp

```diff
@@ -238,12 +239,29 @@ static int waitingThread(void *context) {
     if (res != 0) {
         printf("(%s,%d): TEST FAILURE: waitingThread: DetachCurrentThread() returns: %d\n",
             __FILE__, __LINE__, res);
-        return STATUS_FAILED;
+         exit(STATUS_FAILED);
     }
     if (verbose)
         printf("waitingThread: the thread #%d exits with %d\n",
             indx, exitCode);
-    return exitCode;
+    return 0;
+}
+
+static int waitingThread(void *context) {
+   pthread_t id;
+   int result;
+   pthread_attr_t attr;
+   pthread_attr_init(&attr);
+   pthread_attr_setstacksize(&attr, STACK_SIZE);
```

Comment:
This is failing to build on Windows. Looks like you don't have the needed #include.

Reply:
This cannot build on Windows!